PackageVersionsLatestMinors.g.props

```diff
@@ -0,0 +1,157 @@
+<!--
+***********************************************************************************************
+PackageVersions.g.props
```

Comment:
very small nit: I guess we could just delete this hard-coded part of the template since we're now creating two separate files whose names don't match this. We don't need to rename this for the sake of the PR though since this is blocking other work